### ISSUE TITLE
"View Bobbing" Affects Moons/Stars On Jupiter

### ISSUE THREAD

**Issue by TheRealDannyyy** [MJRLegends/ExtraPlanets#413]
### Description of Issue
I've noticed a minor visual issue with some moons/stars, that can be seen in the skies of Jupiter. The view bobbing effect seems to affect them and make them "bobble" around while walking.

### Steps to Reproduce Issue
1. Enable "view bobbing" setting
2. Visit _Jupiter_
3. Notice moon/star bobbing around

**Crash log/Log (if applicable):** _On request if really required._

**Screenshots (if applicable):** _On request if really required._

**Version of Mod using (PLEASE DONT USE 'Latest' OR 'Newest'):**
- MinecraftForge v14.23.5.2815
- ExtraPlanets-1.12.2-0.4.9
- Galacticraft-Planets-1.12.2-4.0.2.190
- MicdoodleCore-1.12.2-4.0.2.190
- ExtraPlanets-1.12.2-0.4.9
- MJRLegendsLib-1.12.2-1.1.8
- jei_1.12.2-4.15.0.269
- OptiFine_1.12.2_HD_U_E3

(_Server is using exactly the same versions._)

**Where did this bug happen:** Dedicated _Nitrado_ server with connected clients.

**Comment by MJRLegends** [comment]
Does it do it for moon and mars and venus?

**Comment by TheRealDannyyy** [comment]
From what I could see (difficult to reproduce because of #412), the following planets/moons/stars are affected:
(**1** is bobbing the most and **3** the least, not sure why the intensity is different for each one.)
![BobbingExample](https://user-images.githubusercontent.com/15656049/54882040-46978580-4e56-11e9-9081-4ee8a97ef37c.png)

**Comment by MJRLegends** [comment]
Thats not what i meant by does it happen on the moon and venus and mars. I meant the dimensions not the sky. As i believe its the same code they use for there stars as i believe this is a Galacticraft bug too.

**Comment by TheRealDannyyy** [comment]
Sorry for the misunderstanding. I couldn't reproduce it on those planets like I can on Venus. It could be more subtle to the point where it isn't noticeable but I'm not sure. 

Could you maybe look into the code and further identify the cause? I assume that's not a problem for you since you know how all the stuff works. I'll re-open another issue if extra planets is indeed not the cause.

**Comment by MJRLegends** [comment]
I dont think it is the cause as ive seen people report this before. I will look at it though

**Comment by TheRealDannyyy** [comment]
Maybe this will help when investigating. 

While landing, some planets/moons seem to be too closely "attached" to the players view. Seems like they are more like sprites instead of static elements you can see in the sky.

**Comment by TripTripIssueTeller** [comment]
abc abc abc

**Comment by MJRLegends** [comment]
Im not sure how to fix sorry

**Comment by MJRLegends** [comment]
If any one wants to make a pull request to fix please do i will be grateful
